routes/controllers/verifyController.ts

```diff
@@ -1,18 +1,18 @@
 import { Router } from 'express';
 
 //validation, common
-import LoginValidation from '../middleware/login/_validation';
-import JoinValidation from '../middleware/register/_validation';
+import LoginValidation from '../middleware/user/login/_validation';
+import JoinValidation from '../middleware/user/register/_validation';
```

Comment:
```suggestion
import RegisterValidation from '../middleware/user/register/_validation';
```
가독성을 위해 구조 이름을 인용해서 RegisterValidation이 나을 것 같음